src/EventListener/InsecureInstallationListener.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\EventListener;
+
+use Contao\CoreBundle\Exception\IncompleteInstallationException;
+use Contao\CoreBundle\Exception\InsecureInstallationException;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+
+/**
+ * Validates the Installation
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
```

Comment:
It does not make much sense to me to add all authors of the old listener. I'd say you should only add yourself :)